conference.js

```diff
@@ -1766,5 +1766,18 @@ export default {
             // Update the view
             APP.UI.setLocalRaisedHandStatus(raisedHand);
         }
+    },
+    /**
+     * Log event to callstats and analytics.
+     * @param {string} name the event name
+     * @param {int} value the value (it's int because google analitycs supports
+     * only int).
+     * NOTE: Should be used after conference.init
+     */
+    logEvent(name, value) {
+        if(JitsiMeetJS.analytics)
```

Comment:
Not using braces feels like code smell to me; one bad diff and you end up with something breaking seriously because a tiny bit of whitespace got deleted (see the Apple GOTO fail).